packages/main/src/themes/DynamicDateRangePopover.css

```diff
@@ -73,17 +107,17 @@
 }
 
 [ui5-responsive-popover]::part(header) {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: flex-start;
-    height: 2.5rem;
+	display: flex;
+	flex-direction: row;
+	align-items: center;
+	justify-content: flex-start;
+	height: 2.5rem;
 }
 
 .ui5-ddr-header {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: flex-start;
-    height: 2.5rem;
+	display: flex;
```

Comment:
why this identation

Reply:
I replaced it with global replacement to be consistent within the project.